app/src/main/res/layout/comment_card.xml

```diff
@@ -37,27 +38,58 @@
         android:id="@+id/comment_body"
         style="@style/CommentCardBody"
         android:text="@string/A_successfully_funded_project_will_collect_your_pledge_in_its_native_currency"
-        android:visibility="gone"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="@id/avatar"
         app:layout_constraintTop_toBottomOf="@id/avatar" />
 
+    <androidx.appcompat.widget.LinearLayoutCompat
```

Comment:
We decided a while a ago any new UI should use Constraint layout, we are trying to keep flat hierarchies as much as possible.
You can keep the 
`<androidx.appcompat.widget.AppCompatTextView
                android:id="@+id/replies"
                android:layout_weight="1"
                android:text="@string/view_replies"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"/>

            <androidx.appcompat.widget.AppCompatImageView
                android:src="@drawable/ic_mask"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"/>`

and use constaints to position it please

Reply:
+1 to using constraint layout and flattening the view hierarcy

Reply:
done